build.sbt

```diff
@@ -850,6 +851,25 @@ lazy val `scio-extra`: Project = project
   )
   .configs(IntegrationTest)
 
+lazy val `scio-grpc`: Project = project
+  .in(file("scio-grpc"))
+  .settings(commonSettings)
+  .settings(publishSettings)
+  .settings(protobufSettings)
+  .settings(
+    description := "Scio add-on for gRPC",
+    libraryDependencies ++= Seq(
+      "org.apache.beam" % "beam-sdks-java-core" % beamVersion,
```

Comment:
Do we need this dep? It should already be in `scio-core`

Reply:
Almost all modules declare it.
As it is also a direct dependency, I think it is better to keep the declaration explicit